api/envoy/service/auth/v2/attribute_context.proto

```diff
@@ -92,18 +93,20 @@ message AttributeContext {
     // The HTTP URL scheme, such as `http` and `https`.
     string scheme = 6;
 
-    // The HTTP URL query in the format of `name1=value`&name2=value2`, as it
-    // appears in the first line of the HTTP request. No decoding is performed.
+    // This field is always empty, and exists for compatibility reasons. The HTTP URL query is
+    // included in `path` field.
     string query = 7;
```

Comment:
can we enforce empty here? Is it worth it?

Reply:
What do you mean "enforce empty"?  Note that *Envoy* is what populates this, not the AuthService talking to Envoy. I don't believe that code outside of Envoy ever needs to create this.

Reply:
got it thanks.